src/App/Pages/Home.elm

```diff
@@ -15,22 +15,22 @@ import List
 import List.Extra
 
 
-view : Model -> Html Msg
-view model =
+view : Model -> WebData (List Article) -> WebData (List Recipe) -> WebData (List Recipe) -> Html Msg
+view model promotedArticles promotedRecipes recipes =
     div []
-        [ viewPromotedContent model
+        [ viewPromotedContent promotedArticles promotedRecipes
         , viewCurrentMonthIssue model
         , viewCookMenu model
-        , viewRecipes model
+        , viewRecipes recipes
         ]
 
 
-viewPromotedContent : Model -> Html Msg
-viewPromotedContent model =
+viewPromotedContent : WebData (List Article) -> WebData (List Recipe) -> Html Msg
```

Comment:
Depends. But often times it seems that a page is around a single _main_ data type.
But again, depends :)